docs/source/markdown/podman.1.md

```diff
@@ -1,56 +1,52 @@
 % podman(1)
 
 ## NAME
-podman - Simple management tool for pods, containers and images
+podman - simple management tool for pods, containers and images
 
 ## SYNOPSIS
-**podman** [*options*] *command*
+**podman command** [*optional*] *value*
 
 ## DESCRIPTION
-Podman (Pod Manager) is a fully featured container engine that is a simple daemonless tool.
```

Comment:
We want the (Pod Manager) comment here, since we got accused of being a sexist name, and wanted to point out where it comes from.

Reply:
Haha, but I think the purpose of this page is to describe the command podman not Podman

Reply:
@infinteregrets please retain the "Pod Manager" bit here.

Reply:
And I just read the rest of this.  I'd rather not change this paragraph at all until we can get our @fatherlinux to look at it, but he's on paternity leave through the end of the month.

Reply:
Okay, I will fix that. After a short description of the command, should the "Pod Manager" part be put in another paragraph? Or should I just revert it back to the way it was in the first place

Reply:
Yes please revert everything here up to the OPTIONS section.  We can make an issue if you'd like to rework this, but I'd really like to get @fatherlinux 's input as he worked up the original as is on paterinity leave this month.

Reply:
Sure! I have updated the description to the way it was before. Once this is merged, I will open an issue to possibly propose new changes